src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -335,11 +335,12 @@ namespace smt::noodler {
             expr_ref len(m_util_a.mk_numeral(rational(l), true), m);
 
             literal lit(mk_eq(len_str, len, false));
-            ctx.mark_as_relevant(lit);
+            // ctx.mark_as_relevant(lit);
             ctx.mk_th_axiom(get_id(), 1, &lit);
         } else {
             // build axiom 1: Length(a_str) >= 0
-            {
+            { 
+                return; // TODO: fix some day
```

Comment:
Maybe write a quick note what is to be fixed here.

Reply:
ok